radis/test/lbl/test_spectrum.py

```diff
@@ -70,8 +49,16 @@ def test_populations(verbose=True, *args, **kwargs):
     )
     sf.load_databank("HITRAN-CO-TEST")
     sf.misc.export_rovib_fraction = True
-    
-    sf.params.parsum_mode = "full summation"  #won't be default at some point
+    # we test that "tabulation" and "export_population" are incompatible
+    sf.params.parsum_mode = "tabulation"
+    with pytest.raises(ValueError) as err:
```

Comment:
@minouHub just so you know : a cool function `pytest.raises` that will pass only if corresponding Exception is raised.